src/utils/constants.ts

```diff
@@ -51,9 +51,11 @@ export const BasePorts = {
   lnd: {
     rest: 8081,
     grpc: 10001,
+    p2p: 9735,
   },
   clightning: {
     rest: 8181,
+    p2p: 9735,
```

Comment:
I haven't tested but I think having both the lnd and c-lightning ports start at 9735 will cause host port conflicts. This is why I made the REST ports 100 numbers apart.

Reply:
Very good point. They collide. I'll change to 100 apart.
I don' see why though, shouldn't the id+1 set them apart?

https://github.com/jamaljsr/polar/blob/44b9fb479bb3c7603d7ab1aaca7e11f4770c319e/src/utils/network.ts#L90

Reply:
Fixed.

Reply:
> I don't see why though, shouldn't the id+1 in `createLndNode` and `createCLightningNode` set them apart?

They will initially start out correct when the nodes are created. So if you had a network with 4 LN nodes `[LND, c-lightning, LND, c-lightning]`, they will start out with the correct ports `[9735, 9736, 9737, 9738]`. But when you start the network and `getOpenPorts` is called, the port detection is run in groups of the implementations independently. So imagine if port `9737` is unavailable. For LND, `getOpenPortRange` will be called with `[9735, 9737]` and return `[9735, 9738]`. Then `getOpenPortRange` is called for c-lightning nodes with `[9736, 9738]` and will return `[9736, 9738]` since both of those ports are currently open. We now have a conflict on port `9738` when docker tries to start the containers. This likely has a very rare chance of occurring, but if we can easily avoid it by adding the gap of 100, why not?

Reply:
Ah right, makes sense! Thank you for the explanation.

Reply:
After these discussions, I am thinking it may be a good idea in the future to remove the `+ id` when creating nodes. This isn't really doing anything meaningful other than avoiding a call to `dockerService.saveComposeFile` if all of the initially set ports are available on startup. I now see that it causes some confusion when reading the code. It should work fine by just setting these values to the BasePorts then let `getOpenPorts` reassign all the values when the node is started. 

I am not suggesting that you should change anything in this PR, I'm just mentioning it as a thought I have.

Reply:
@jamaljsr Yes I have been kind of thinking in the same lines, the `+ id` is kind of confusing, it's better to make it more clear that `getOpenPorts` is ultimately responsible for fixing any conflicts.  
But yes this could be addressed at a later point.